src/gpuarray_buffer_cuda.c

```diff
@@ -1305,6 +1305,10 @@ static int cuda_callkernel(gpukernel *k, unsigned int n,
     }
 
     switch (n) {
+    case 0:
+      ctx->err = cuLaunchKernel(k->k, 1, 1, 1, 1, 1, 1, shared,
+                                ctx->s, args, NULL);
+      break;
```

Comment:
@abergeron I amended the commit, that file is no longer touched. However, I do think it made sense. The pattern is pretty clear:

- 3-dims is a volume XYZ
- 2-dims is a matrix XY with Z=1
- 1-dims is a vector X with Y=Z=1
- 0-dims is a scalar with X=Y=Z=1